### ISSUE TITLE
Finicky opens link in currently open browser rather than designated one in Big Sur

### ISSUE THREAD

**Issue by shahrviral** [johnste/finicky#148]
I was currently able to enable Finicky on BigSur by using the defaultbrowser homebrew package, however the rules only get enforced if all the browsers are running currently. For example, if I have a rule which says open github.com in Firefox Developer Edition, it will only open in FDE if FDE is running. It no longer launches FDE. If FDE is not running, it will launch in the default browser. Am I configuring this wrong?

```javascript
module.exports = {
  defaultBrowser: 'Firefox',
  handlers: [
    {
      match: /github.com\/Acme/,
      browser: 'Firefox Developer Edition',
    },
],
};
```

Test URL: https://github.com/Acme/my_repo

Thanks,
shahrviral

**Comment by johnste** [comment]
Config looks ok to me, and I can't reproduce the issue (though I am not using the homebrew package). Not sure what I can do about this right now unfortunately. I'm trying to find the solution for #146, maybe that will solve this issue as well.

**Comment by shahrviral** [comment]
I appreciate you taking a look, yeah perhaps they are related. I'll wait and keep an eye on #146

**Comment by johnste** [comment]
Mistakenly closed this issue instead of #146, will keep this open for the time being.

**Comment by johnste** [comment]
Release 3.1.3 fixes #146 so might be worth testing this again

**Comment by shahrviral** [comment]
Fixed in this release, thanks 👍
